recipes/gperf/all/conanfile.py

```diff
@@ -47,15 +48,14 @@ def _configure_autotools(self):
                             "STRIP=:",
                             "AR={}/build-aux/ar-lib lib".format(cwd),
                             "RANLIB=:"])
-
-            self._autotools = AutoToolsBuildEnvironment(self, win_bash=win_bash)
-            self._autotools.configure(args=args)
+            elif self.settings.compiler == "gcc" and self.settings.os_build == "Windows":
+                args.append("LDFLAGS=-static -static-libgcc")
+            self._autotools.configure(args=args, configure_dir=self._source_subfolder)
         return self._autotools
 
     def _build_configure(self):
-        with tools.chdir(self._source_subfolder):
```

Comment:
Removing this line and adding `configure_dir=self._source_subfolder` doesn't work for me on Windows with VS

Reply:
@madebr I have undone this change, but took over the rest. Is this fine for MinGW on Linux?

I tested with GCC9/Linux and VS2019/Windows

Reply:
Yes, the only error were missing symbols when testing.
I've cleaned up the pr a bit. Maybe you can extract some parts of it.